examples/logicalclusters/README.md

```diff
@@ -0,0 +1,68 @@
+# LogicalCluster to workspace hierarchy Example
+
+This example demonstrates how to resolve a `LogicalCluster` to a `Workspace` using the KCP API and
+walk up the workspace hierarchy tree, displaying parent workspace information. This might be useful
+if one stores metadata in parent workspace and wants to access it.
+
+But one needs to be careful when doing this, as walking up the hierarchy can be expensive if done
+frequently, and workspaces are located on different shards, so each step up the hierarchy might
```

Comment:
_might_ require a network call? In what cases does fetching "root:foo:bar", as the parent of "root:foo:bar:blub", *not* cause a network call?

Also, will this work with shards only, or should we mention that in this case, the URL needs to point to a front-proxy? IIRC localproxies cannot redirect to other shards.